lib/iris/tests/unit/coords/test_Cell.py

```diff
@@ -120,22 +121,87 @@ def test_PartialDateTime_other(self):
 
 
 class Test_contains_point(tests.IrisTest):
```

Comment:
No-one has told you this yet, but we are gradually migrating over to PyTest-style tests. Iris already fully supports these styles of test, so this class is a great opportunity for using [`parametrize`](https://docs.pytest.org/en/stable/how-to/parametrize.html#pytest-mark-parametrize-parametrizing-test-functions).

Reply:
Turns out sticking with `unittest`-style this was a conscious decision with @stephenworsley for keeping the file consistent, which I can agree with.